tcks/microprofile-opentracing/pom.xml

```diff
@@ -0,0 +1,165 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-tck-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-tck-microprofile-opentracing</artifactId>
+    <name>Quarkus - TCK - MicroProfile OpenTracing</name>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <suiteXmlFiles>
```

Comment:
If there's no exclusions in the tck suite file, can it be removed and the tests are just scanned from the dependencies?

Reply:
After I split this into two modules, I can remove the suite file from the `rest-client` module, but not from the `base` module (in that case I'm getting some exceptions from Arc). Not sure exactly why, apparently some initialization code is skipped if I do that